rtl/eclrtl/rtlrecord.cpp

```diff
@@ -575,7 +575,7 @@ size32_t RtlRecord::calculateOffset(const void *_row, unsigned field) const
         unsigned numOffsets = getNumVarFields() + 1;
         size_t * variableOffsets = (size_t *)alloca(numOffsets * sizeof(size_t));
         RtlRow sourceRow(*this, nullptr, numOffsets, variableOffsets);
-        sourceRow.setRow(_row, field);
+        sourceRow.setRow(_row, field+1);
```

Comment:
It's required if calling getOffset (which will asset without it). But perhaps the assert is wrong?

Reply:
(I added the +1 because I hit the assert - but I don't recall the exact details about whether I was trying to get the offset or the size at the time.)